src/backend/routers/case_manager.ts

```diff
@@ -28,10 +51,11 @@ export const case_manager = router({
         first_name: z.string(),
         last_name: z.string(),
         email: z.string().email(),
+        grade: z.string(),
```

Comment:
```suggestion
        grade: z.number(),
```

(then don't need the `Number()` cast below)

Reply:
I think the issue here is that the grade is input as a text, so we would need to convert it to Number either here or on the frontend. Unless setting z.number() automatically converts strings to numbers?

Reply:
I would recommend converting on the frontend. Alternatively you can do `z.coerce.number()` to automatically parse strings into numbers.

Reply:
We converted all grades to numbers and will only accept numbers in the frontend.